src/index.js

```diff
@@ -19,6 +19,7 @@ import CoreUiApi from './modules/ui';
 import UsersApi from './modules/users';
 import ConversationsApi from './modules/conversations';
 import MyConversationsApi from './modules/myConversations';
+import ContactsApi from './modules/contacts.js';
```

Comment:
Would you mind changing from `contacts` to `externalContacts` for consistency with the public api?  Sorry!  I know this will pretty much touch everything in the PR (filename, module name, examples, test).  Hope it's not too much of a PITA.

Looks like I'm using camel case for most things